src/main/java/com/microsoft/aad/msal4j/AuthorizationRequestUrlParameters.java

```diff
@@ -72,10 +72,20 @@ private AuthorizationRequestUrlParameters(Builder builder){
 
         // Optional parameters
         if(builder.claims != null){
-            String claimsParam = String.join(" ", builder.claims);
+            String claimsParam = JsonHelper.mergeJSONString(builder.claims);
             requestParameters.put("claims", Collections.singletonList(claimsParam));
         }
 
+        if(builder.clientCapabilities != null){
+            String capabilitiesParam = JsonHelper.mergeJSONString(builder.clientCapabilities);
+            if (requestParameters.containsKey("claims")) {
+                String mergedClaimsAndCapabilities = JsonHelper.mergeJSONString(String.valueOf(requestParameters.get("claims").get(0)), capabilitiesParam);
+                requestParameters.replace("claims", Collections.singletonList(mergedClaimsAndCapabilities));
+            } else {
+                requestParameters.put("claims", Collections.singletonList(capabilitiesParam));
```

Comment:
Please note that claim and client capabilities should be sent to /token endpoint as well.

Reply:
![image](https://user-images.githubusercontent.com/12273384/81280088-e78fd280-904f-11ea-8040-b2d36d844910.png)